deployment/ccip/changeset/v1_6_2/cs_deploy_usdc_token_pools.go

```diff
@@ -165,10 +165,10 @@ func deployUSDCTokenPoolContractsLogic(env cldf.Environment, c DeployUSDCTokenPo
 		case shared.HybridLockReleaseUSDCTokenPool:
 			deployErr = deployHybridLockReleaseUSDCTokenPool(env.Logger, chain, newAddresses, poolConfig, chainState, router.Address())
 		default:
-			return cldf.ChangesetOutput{}, fmt.Errorf("failed to deploy %s on %s: %w", poolConfig.PoolType, chain, err)
+			return cldf.ChangesetOutput{}, fmt.Errorf("failed to deploy %s on %s: %w", poolConfig.PoolType, chain, deployErr)
```

Comment:
seems like we wanted to log `deployErr` here instead of `err`

Reply:
Looks like there is no error for this case
```suggestion
			return cldf.ChangesetOutput{}, fmt.Errorf("failed to deploy %s on %s: unknown pool type", poolConfig.PoolType, chain)
```